src/themes.ts

```diff
@@ -4,23 +4,25 @@ import { sortByKey } from "./sort";
 import { getVariants } from "./variants";
 
 type ThemeRaw = {
+  icon?: string;
   id: string;
-  title: string;
-  variants: string;
+  introduction: string | undefined;
```

Comment:
introduction? : string
```suggestion
  introduction: string | undefined;
```

Reply:
Le `| undefined` c'est pour le cas ou il est null si j'ai bien compris (ce qui arrive), mais normalement il ne devrait jamais être undefined. Mais en fait je suis obligé de rajotuer le `?` quand même sinon typescript rale parce que dans le type `theme` j'ai mis : `introduction?: string;` vu que là du coup il peut être undefined à cause de l'assignement `introduction: theme.introduction`. Mais au moins c'est secure comme ça :trollface: 
Je sais pas si j'ai tout bien compris

Reply:
take a look here 
- https://www.typescriptlang.org/docs/handbook/basic-types.html#null-and-undefined
- https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties